.github/workflows/codeql.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       #   uses: actions/setup-example@v1
 
       - name: Initialize CodeQL
-        uses: github/codeql-action/init@v3
+        uses: github/codeql-action/init@v4
```

Comment:
try also updating line 82 to codeql-action/analyze@v4 

I think the version mismatch is causing the code ql failure.